pkg/network/setup/network.go

```diff
@@ -38,7 +38,7 @@ type VMNetworkConfigurator struct {
 }
 
 type configStateUnplugger interface {
-	UnplugNetworks(vmi *v1.VirtualMachineInstance, cleanupFunc func(string) error) error
+	UnplugNetworks(vmi *v1.VirtualMachineInstance, cleanupFunc func(string, int) error) error
```

Comment:
nit: Please consider adding the parameter names of the cleanup function.

Reply:
done.